pkgs/development/python-modules/ultralytics_thop/default.nix

```
@@ -0,0 +1,44 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, numpy
+, torch
+, pythonOlder
+, setuptools
+, wheel
+}:
+
+buildPythonPackage rec {
+  pname = "ultralytics_thop";
+  version = "2.0.8";
+  format = "pyproject";
+
+  disabled = pythonOlder "3.8";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "fd098468a5a50609c28906c557240c0d1603175c5420843de5ebefd7c83376ab";
+  };
+
+  nativeBuildInputs = [
+    setuptools
+    wheel
+  ];
+
+  propagatedBuildInputs = [
+    numpy
+    torch
+  ];
+
+  pythonImportsCheck = [ "thop" ];
+
+  doCheck = true;
+
+  meta = with lib; {
+    description = "Profile PyTorch models for FLOPs and parameters.";
+    homepage = "https://github.com/ultralytics/thop";
+    licenses = with licenses; [ agpl3 ];
```

```suggestion
    licenses = licenses.agpl3Plus;
```
https://github.com/ultralytics/thop/blob/87f13d1b7bb706c403d8b7ca87a05ad8c9d1ffca/pyproject.toml#L45